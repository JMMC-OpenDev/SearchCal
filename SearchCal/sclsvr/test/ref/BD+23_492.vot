version https://git-lfs.github.com/spec/v1
oid sha256:7dea923acbd54caa7ef0639a4ff6d41324773fd408aea9036ac851b025158dad
size 226638
