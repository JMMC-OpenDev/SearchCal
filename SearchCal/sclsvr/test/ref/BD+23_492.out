version https://git-lfs.github.com/spec/v1
oid sha256:216662e563f03620579be9397bd4abe60d5fd29f37f08228187c4b899c50efb7
size 409388
